src/mobility/components/MobilityDistance.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {Walk} from '@atb/assets/svg/mono-icons/transportation';
+import {ThemeText} from '@atb/components/text';
+import {useHumanizeDistance} from '@atb/utils/location';
+import {StyleSheet, useTheme} from '@atb/theme';
+
+type Props = {
+  distance: number | undefined;
+  style?: StyleProp<ViewStyle>;
+};
+export const MobilityDistance = ({style, distance}: Props) => {
```

Comment:
Why do we need this as specific component for mobility? We should try to minimize duplication. The only differences I can see is the margins and paddings. Could this be achieved by adding som options to the standard `<WalkingDistance />` component instead?

Reply:
can I do something like this instead?

```
type Props = {
  distance: number | undefined;
  style?: StyleProp<ViewStyle>;
  isMobility: boolean;
};
export const WalkingDistance = ({style, distance, isMobility}: Props) => {
  const sheetStyles = useSheetStyle();
  const humanizedDistance = useHumanizeDistance(distance);
  const {theme} = useTheme();

  const labelStyle = isMobility ? sheetStyles.mobilityLabel : sheetStyles.distanceLabel;
  const iconStyle = isMobility ? sheetStyles.icon : {}

  if (!humanizedDistance) return null;

  return (
    <View style={[style, labelStyle]}>
      <ThemeIcon svg={Walk} fill={theme.text.colors.secondary} style={iconStyle}/>
      <ThemeText type="body__secondary" color="secondary">
        {humanizedDistance}
      </ThemeText>
    </View>
  );
};
```